homeassistant/components/device_tracker/googlehome.py

```diff
@@ -89,5 +89,6 @@ def __init__(self, hass, config):
                 devices[uuid]['btle_mac_address'] = device['mac_address']
                 devices[uuid]['ghname'] = ghname
                 devices[uuid]['source_type'] = 'bluetooth'
+                devices[uuid]['btle_name'] = device['name']
```

Comment:
Most devices will not have this property resulting in blank attributes.
I suggest checking if it has value before adding it.

```suggestion
                if device['name']:
                    devices[uuid]['btle_name'] = device['name']
```

Reply:
Thanks for that :smile: 

I was lucky and all the devices I checked had names :fearful: